modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordConfigAPI.kt

```diff
@@ -30,6 +30,7 @@ class InvalidConfigException(path: String, reason: String) : MirrordError("faile
  * @throws InvalidConfigException if config cannot be found or parsed.
  */
 fun isTargetSet(config: String?): Boolean {
+    config ?: return false
```

Comment:
Actually not really necessary for this fix, just seemed appropriate to me. Does it make sense or should I remove it?

Reply:
That's cool :+1: